package.json

```diff
@@ -79,12 +79,14 @@
     "react": "^16.8.3",
     "react-dnd-test-backend": "^7.4.4",
     "react-test-renderer": "^16.8.5",
+    "regenerator-runtime": "^0.13.2",
     "rollup": "^1.2.0",
     "rollup-plugin-json": "^3.1.0",
     "rollup-plugin-typescript2": "^0.19.2",
     "sinon": "^7.2.7",
     "storybook-readme": "^5.0.2",
     "styled-components": "^4.1.3",
+    "three": "^0.104.0",
```

Comment:
We should probably not use this, but rather use { THREE } from the 3d-viewer. At least use the same version (0.98.0) as the 3d-viewer. We have seen some problems using three.js (ref https://github.com/cognitedata/3d-viewer/issues/1002 that may not be visible for all) in combination.

So the recommended thing to do is `import { Cognite3DViewer, THREE } from '@cognite/3d-viewer';` and use that `THREE`.

Reply:
We're actually using `{THREE} from @cognite/3d-viewer` package, this one dependency related only to Stroybook test case, where we importing OBJLoader to mock model adding to viewer.

Reply:
@mrtraser Can't we use the `THREE` from 3d-viewer here as well?

Reply:
@anderscognite He needs this to use the `OBJLoader` for mocking in storybook. So will not be exposed to the end-user. Our `THREE` object doesn't expose any OBJLoader